exopy/app/log/manifest.enaml

```diff
@@ -194,9 +201,22 @@ enamldef LogManifest(PluginManifest):
             id = 'exopy.app.logging'
             sync_members = ['handler_ids', 'filter_ids', 'gui_model']
 
+    Extension:
+        id = 'menus'
+        point = 'enaml.workbench.ui.actions'
+        ActionItem:
+            path = 'tools/log'
+            label = 'Open Log'
+            command = 'exopy.app.logging.display_rotating'
+
+
     Extension:
         id = 'commands'
         point = 'enaml.workbench.core.commands'
+        Command:
+            id = 'exopy.app.logging.display_rotating'
+            description = DISPLAY_DESCRIPTION
+            handler = make_handler(PLUGIN_ID, 'display_rotating')
```

Comment:
I propose to rename this method to "display_current_log", since we do not really care it is a rotating one, but we want the one currently in use. Please update the id too.